action.yml

```
@@ -6,10 +6,12 @@ branding:
 inputs:
   auth-token:
     description: 'If you are working in a private repo, you will need to specify the [GitHub secrets token](https://docs.github.com/en/actions/reference/authentication-in-a-workflow)'
+    default: ${{ github.token }}
     required: false
   workflow-run-id:
     description: 'The ID of the workflow run that has the artifacts to download.'
-    required: true
+    default: ${{ github.event.workflow_run.id }}
+    required: false
```

```suggestion
    required: true
```
Even though you are passing a default, it's still a required value as an input. Otherwise the action will break.